README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 01
+# HW01
+## Deven Mukkamala, Biography
+
+![Deven Mukkamala](https://www.legion.org/sites/legion.org/files/legion/boys-nation-delegates/20170721_Michigan%20Federalist%20Deven%20Mukkamala_7501.jpg)
+
+My name is **Deven Mukkamala**. I am a *first-year* in _the College_ studying _Economics_ and _Philosophy_. I was born in **Oak Park**, but I moved to **Flint, Michigan** when I was young.
+
+A few of my hobbies of mine are: 
+
+* Basketball (The Chicago Bulls)
+* Reading (Non-fiction)
+* French Horn 
+* Debate
+
+If I had to rank my favorite philosophers based on _writing style_, rather than _beliefs_, I would say:
+
+1. Jean-Jacques Rousseau
```

Comment:
You don't have to increment the numbers yourself. When the document renders, this automatically happens.

```
1. Jean-Jacques Rousseau
1. Thomas Hobbes 
1. Plato
```

becomes

1. Jean-Jacques Rousseau
1. Thomas Hobbes 
1. Plato